js/admin.vue

```diff
@@ -273,11 +274,11 @@ export default {
                     this.$nextTick(cb);
                 }
             } else {
-                require(['./views/' + name + '.vue'], function(options) {
-                    self.$options.components[name] = Vue.extend(options);
-                    self.view = name;
+                require(['./views/' + name + '.vue'], (options) => {
+                    this.$options.components[name] = Vue.extend(options);
```

Comment:
Are you sure about this ? Last time I checked, ``require`` override the ``this`` scope, even on arrow functions.